src/main.ts

```diff
@@ -121,6 +162,15 @@ export class Minifier {
     return Minifier.reservedJSKeywords.hasOwnProperty(str);
   }
 
+  renameProperty(name: string): string {
+    if (!this.renameMap.hasOwnProperty(name)) {
+      let newName = this.generateNextPropertyName(this.lastGeneratedPropName);
```

Comment:
Done (though this adds three places before `return` statements where assignment needs to be done)